Source/PeleLMeX_Advance.cpp

```diff
@@ -120,6 +120,13 @@ PeleLM::Advance(int is_initIter)
     computeSootSource(AmrOldTime, m_dt);
   }
 #endif
+#ifdef PELELM_USE_RAD
```

Comment:
shouldn't this also be PELELMEX?

Reply:
This is following the style of all PELEMP macros: PELELM_USE_SPRAY and PELELM_USE_SOOT.